src/client/automation/playback/type/index.js

```diff
@@ -156,8 +167,12 @@ export default class TypeAutomation {
     }
 
     _typingStep () {
-        this.currentKeyCode  = getKeyCode(this.text.charAt(this.currentPos));
-        this.currentCharCode = this.text.charCodeAt(this.currentPos);
+        var char = this.text.charAt(this.currentPos);
+
+        this.currentKeyCode       = getKeyCode(char);
+        this.currentCharCode      = this.text.charCodeAt(this.currentPos);
+        this.currentKey           = this.currentKeyCode === 13 ? 'Enter' : char;
```

Comment:
and why we check only `enter` here. Maybe we should modify all keys from `keyProperty` in utils/key-maps.js ?